package.json

```diff
@@ -53,63 +53,64 @@
     "Base64": "^1.0.0",
     "array.of": "^0.1.1",
     "audio-context-polyfill": "^1.0.0",
-    "babel-core": "^6.13.2",
+    "babel-core": "^6.23.1",
     "babel-preset-es2015": "^6.1.18",
     "blocked": "^1.1.0",
     "cli-color": "^1.1.0",
     "convert-source-map": "^1.3.0",
     "denodeify": "^1.2.1",
-    "dotenv": "^2.0.0",
-    "exists-sync": "0.0.3",
-    "express": "^4.4.4",
+    "dotenv": "^4.0.0",
+    "exists-sync": "0.0.4",
+    "express": "^4.14.1",
     "fast-stats": "0.0.3",
     "from2-string": "^1.1.0",
     "glob": "^7.1.1",
     "graceful-fs": "^4.1.10",
-    "graphite": "0.0.6",
-    "handlebars": "^4.0.5",
+    "graphite": "0.0.7",
+    "handlebars": "^4.0.6",
     "html5shiv": "^3.7.3",
-    "intersection-observer": "^0.2.0",
+    "intersection-observer": "^0.2.1",
     "intl": "^1.2.5",
-    "js-polyfills": "^0.1.22",
+    "js-polyfills": "^0.1.32",
```

Comment:
Yep, I saw this.  I wasn't able to run the tests locally as I don't have the browserstack credentials and there are three sets in lastpass (not clear which one to use) and when I found out which one to use, it didn't work.  So it would be helpful to fix that.

I'll look into trying to fix this, but if not I'll roll back that dependency.